docker/activemq/README.md

```diff
@@ -1,18 +1,76 @@
 ActiveMQ-docker
 ============
-Dockerfile source for ActiveMQ [docker](https://docker.io) image.
+Dockerfile source for ActiveMQ [docker](https://docker.io) image..
 
-# Quick Start for ActiveMQ 5.x
+## Upstream
+This source repo was originally copied from:
 
-You can launch the image using the `docker` command line:
+
+## Disclaimer
+This is not an official Google product.
+
+## About
+This image contains an installation of ActiveMQ
+
+For more information, see the
+[Official Image Marketplace Page](https://console.cloud.google.com/marketplace/product/bitnami-launchpad/activemq).
+
+### Prerequisites
+
+Configure [gcloud](https://cloud.google.com/sdk/gcloud/) as a Docker credential helper:
+
+```shell
+gcloud auth configure-docker
+```
+### Pull command
+
+```shell
+docker -- pull marketplace.gcr.io/google/activemq5
+```
+## Table of Contents
+
+ [Using Docker](#using-docker)
+  * [Run a  server](#run-a-activemq-server-docker)
+    * [Start an activemq instance](#start-a-activemq-instance-docker)
+* [References](#references)
+  * [Ports](#references-ports)
+
+# Using Docker
+
+Consult [Marketplace container documentation](https://cloud.google.com/marketplace/docs/container-images)
+for additional information about setting up your Docker environment.
 
 ## Create and set ownership of `data/` directory to `activemq` user.
 ```shell
 mkdir data/
 chown 1000:1000 data/
 ```
+## Start an activemq instance
+
+Use the following content for the `docker-compose.yml` file, then run `docker-compose up`.
+
+```yaml
+version: '2'
+services:
+  activemq:
+    container_name: activemq
+    image: marketplace.gcr.io/google/activemq5
+    environment:
+      "ACTIVEMQ_ADMIN_LOGIN": "setyourdesiredlogin"
+      "ACTIVEMQ_ADMIN_PASSWORD": "example-password"
+    ports:
+      - '5672:5672'
+      - '61613:61613'
+      - '1883:1883'
+      - '61614:61614'
+      - '8161:8161'
+    volumes:
+      - ./data/:/opt/activemq/data
+
+  ```
+  
+Or you can use `docker run` directly:
 
-## Run `docker run` command.
 ```shell
 docker run -e ACTIVEMQ_ADMIN_PASSWORD="setyourdesiredpassword" \
```

Comment:
I'd format the command as below: 

```shell 
docker run \
    -e ACTIVEMQ_ADMIN_LOGIN="example-username" \
    -e ACTIVEMQ_ADMIN_PASSWORD="example-password" \
    -it --rm \
    marketplace.gcr.io/google/activemq5
```

* Better environment variable displays
* Keep consistent with the example we have above